### DISCUSSION TITLE
KUDOS TO the team

### DISCUSSION THREAD

**Discussion by surfingknowledge** [microsoft-search/pnp-modern-search#1933] [Show and tell]
**Kudos to** @FranckyC @wobba and others who are contributing to this solution. I just wanted to take a moment to give you all  the credit you deserve. I built fantastic solution using your product and all my clients are happy and it's well accepted across my org, I also appreciate the team's support and for answering all incoming questions and for trying to solve all unforeseen issues.  You made our life easy and took SP/M365 search capability to another level , I like it most b/s it's very extensible and customizable to meet different customer's  requirements. 

Thank you again, @FranckyC . Keep it up.

It's been a while I couldn't find the person in the discussion board but he had same issue i was having somehow I was able to solve it. IDK what I did but it's solved and sorry I couldn't get back to him to help.

**Comment by wobba** [comment]
Thank you for the kind words and glad to hear the solution solves your customers scenarios 👍

**Reply by FranckyC** [reply]
Thanks @surfingknowledge ! Not so often we get encouraging messages

**Reply by VerlindenEAM** [reply]
Indeed great work for this heavily used powerful solution!

KUDOS to @wobba @FranckyC and rest of the Team!
